cmd/weak-key-search/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/goodkey"
+	blog "github.com/letsencrypt/boulder/log"
+)
+
+type certInfo struct {
+	serial string
+	der    []byte
+}
+
+var limit = 1000
```

Comment:
It might be worth plumbing this through as a CLI option in the event perf testing indicates we have to slide this in either direction.

Reply:
Done.